tests/test_phase_psf.py

```diff
@@ -254,67 +282,68 @@ def test_phase_psf_reset():
     # Test frozen AtmosphericScreen first
     atm = galsim.Atmosphere(screen_size=30.0, altitude=10.0, speed=0.1, alpha=1.0, rng=rng)
     aper = galsim.Aperture(diam=1.0, lam=500.0)
-    wf1 = atm.wavefront(aper)
-    atm.advance()
-    wf2 = atm.wavefront(aper)
+    wf1 = atm.wavefront(aper.u, aper.v, None)
+    atm._seek(1.0)
+    wf2 = atm.wavefront(aper.u, aper.v, None)
```

Comment:
Good catch.  I purposely left the possibility of using `None` for the `t` argument undocumented, so this might have been okay, but it was easy enough to switch the test to use `_wavefront` instead of `wavefront` and remove the special case logic out of the latter.  So I think the public API methods should be okay now.

As for whether to make wavefront public or not, I think it's nice to include for its own sake, even if it's not simple to convert a wavefront result into a PSF.

Looking through this also made me realize that I should document the fact that the `t` and `theta` arguments to `wavefront` and `wavefront_gradient` don't do anything at all for `OpticalScreen`s, so I also edited those docstrings.